lib/screens/activity_log_screen.dart

```diff
@@ -37,6 +39,7 @@ class _ActivityLogScreenState extends State<ActivityLogScreen> {
       ));
     }
     return ListView.separated(
+      padding: EdgeInsets.symmetric(vertical: 8),
```

Comment:
Is this to give the Entries some more spacing?

Reply:
yes